src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -130,9 +133,12 @@ public partial struct JsonDocumentOptions
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
+        public bool IsImmutable { get { throw null; } }
         public System.Text.Json.JsonElement this[int index] { get { throw null; } }
         public System.Text.Json.JsonValueKind ValueKind { get { throw null; } }
         public System.Text.Json.JsonElement Clone() { throw null; }
+        public static System.Text.Json.JsonNode DeepCopy(System.Text.Json.JsonDocument jsonDocument) { throw null; }
```

Comment:
Why did we decide to take the static approach? technically we could have just made them instance methods and that way you don't need two overloads right?

Reply:
That's just what I understood from my notes. I can make them instance methods.

Reply:
I don't think we want bidirectional surface area mappings... cloning a JsonElement into a JsonNode tree feels like a JsonNode operation ("I can build me from that"), not a JsonElement operation... since there's no instance to apply to once that's true, it has to be static.

Reply:
So I should keep them static and move them to `JsonNode`?

Reply:
Yeah, static methods on JsonNode is what I would expect.